### PR TITLE
fix: Recover configuration from DB on restart

### LINKED ISSUE #85: Webconsole loses its state after a restart
Webconsole currently loses its GRPC state when restarted for any reasons. If a subscriber, device group and network slice is properly configured and webconsole is restarted, all the NFs depending on webui will enter a tight loop trying to connect to it, with the connection status being idle.

This can also prevent the network for working at all if for example webconsole is restarted, and then the AMF is restarted. The AMF will not have any configuration, waiting in a loop for webconsole to provide it.

The only way to recover webconsole at that point is to HTTP POST a network slice or device group.

### PR DESCRIPTION
**Pull request by ghislainbourgeois**
This change makes webconsole check its database on startup for existing network slices or device groups instead of waiting for new configuration to be POSTed.

This was tested with a full deployment of SD-Core, running a simulation with gnbsim. After a successful simulation, I took a backup of the database, removed the entire infrastructure, and redeployed mongo, restoring from the backup, then redeploying SD-Core and running a new successful simulation with gnbsim.

fixes #85

### CODE REVIEW

**Review by saltiyazan — APPROVED** [review]

**Review by gatici — APPROVED** [review]

**Comment by gab-arrobo** [comment]
test this please

**Comment by gab-arrobo** [comment]
test this please

File: proto/server/configEvtHandler.go [hunk 1]
```diff
@@ -191,6 +194,15 @@ func handleNetworkSlicePost(configMsg *configmodels.ConfigMessage, subsUpdateCha
 	rwLock.Unlock()
 }
 
+func firstConfigReceived() bool {
+	if len(getDeviceGroups()) > 0 {
+		return true
+	} else if len(getSlices()) > 0 {
+		return true
+	}
+	return false
```
Comment (gab-arrobo) [inline proto/server/configEvtHandler.go:203]: What about this change?
```suggestion
	return len(getDeviceGroups()) > 0 || len(getSlices()) > 0
```

Comment (ghislainbourgeois) [reply proto/server/configEvtHandler.go]: Applied the suggestion, thank you!

**Comment by gab-arrobo** [comment]
@ghislainbourgeois, please resolve conflict.

**Comment by ghislainbourgeois** [comment]
> @ghislainbourgeois, please resolve conflict.

@gab-arrobo I resolved the conflict.

File: proto/server/configEvtHandler.go (hunk 1 above)
Comment (gab-arrobo) [reply proto/server/configEvtHandler.go:215]: It seems that this change got reverted. Or is it intentional?

**Review by thakurajayL — APPROVED** [review]

**Review by gab-arrobo — APPROVED** [review]
+1

**Comment by dafATonf** [comment]
test this please

**Pull request merged**